js/index.js

```diff
@@ -1,7 +1,53 @@
-// Iteration 1: Names and Input
+// Iteration 1
+const hacker1 = 'Carlos'
+const hacker2 = 'Esteban'
 
+console.log(`The driver name is ${hacker1}`)
+console.log(`The navigator name is ${hacker2}`)
 
-// Iteration 2: Conditionals
+// Iteration 2
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length}`)
+} else if (hacker2.length > hacker1.length) {
+    console.log(`It seems the navigator has the longest name, it has ${hacker2.length}`)
+} else if (hacker2.length === hacker1.length) {
+    console.log(`Wow, you have both have equally long names, ${hacker2.length}`)
+}
 
+// Iteration 3
+console.log(hacker1.split('').join(' ').toUpperCase())
```

Comment:
esto esta bien por que no especificaba que tenian que hacer pero la mejor practica es hacer una funcion y consologearla.

`let str = "";
for(i = 0; i < hacker1.length; i++){
  if(i < hacker1.length-1){
    str += `${hacker1[i].toUpperCase()} `
  } else {
    str += `${hacker1[i].toUpperCase()}`;
  }
}`

En este caso se crea una nueva variable, se recorre el nombre del hacker en la funcion y se van agregando las letras capitalizadas con un espacio a la variable, y si ya se recorrieron todas las letras entonces ya no se agrega el espacio.